trie/verkle.go

```diff
@@ -275,7 +275,9 @@ func (trie *VerkleTrie) Commit(_ bool) (common.Hash, *NodeSet, error) {
 	}
 	batch.Write()
 
-	return common.BytesToHash(nodes[0].CommitmentBytes[:32]), nil, nil
+	// Serialize root commitment in compressed form, and only the root commitment
+	rootH := root.Hash().BytesLE()
```

Comment:
Uhm, note that `root.hash()` returns a `Fr` so technically there's no compressed or uncompressed option.